style.css

```diff
@@ -407,3 +427,326 @@ ul {
+    justify-content: space-evenly;
+    align-items: center;
+    margin: 16px 142px 16px 0;
+    padding: 0;
+  }
+
+  .header {
+    height: 72px;
+    width: auto;
+  }
+
+  .logo {
+    height: 70px;
+    width: 70px;
+    margin: 5px 0 5px 142px;
+  }
+
+  .victor {
+    display: none;
+  }
+
+  .top {
+    display: flex;
+    flex-direction: row;
+    height: 798px;
+    padding: 0;
+    width: auto;
+  }
+
+  .content-page1 {
+    margin: 190px 0 10px 143px;
+  }
+
+  .hfirst {
+    font-size: 72px;
+    line-height: 82px;
+    width: 566px;
+    height: 160px;
+    margin: 0 0 24px 0;
+  }
+
+  .hsecond {
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    width: 566px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    margin: 0 0 24px 0;
+  }
+
+  .paragraph {
+    width: 566px;
+    height: 123px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .icon {
+    height: 20px;
+    width: 20px;
+  }
+
+  .social {
+    width: 20px;
+    height: 200px;
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: column;
+    position: absolute;
+    margin: 351px 100px 341px 36px;
+  }
+
+  .stetescope-icone {
+    display: flex;
+    position: absolute;
+    height: 550px;
+    width: auto;
+    top: 250px;
+    right: 0;
+  }
+
+  .title-work {
+    border-bottom: none;
+    color: #091e42;
+    padding-bottom: 24px;
+    margin: 135px 24px 114px 0;
+  }
+
+  .recent-work {
+    padding: 0%;
+    margin: 0%;
+    margin: 0 143px 130px 140px;
+  }
+
+  .projects {
+    height: auto;
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+  }
+
+  .pro-img {
+    width: auto;
+    height: 448px;
+  }
+
+  .project-1 {
+    display: flex;
+    flex-direction: column;
+    width: 448px;
+    height: 348px;
+    margin: 0 0 0 24px;
+  }
+
+  .stories {
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin: 0%;
+  }
+
+  .about-stories {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .cards {
+    display: flex;
+    color: #091e42;
+    border-color: #091e42;
+  }
+
+  .cards li {
+    border: 1px solid rgba(137, 147, 164, 1);
+  }
+
+  .mobile-projects {
+    display: none;
+  }
+
+  .desktop-projects {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    width: 100%;
+  }
+
+  .second-project {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: auto;
+    height: 435px;
+    margin: 24px 24px 24px 0;
+    padding: 0%;
+  }
+
+  .link-project2 {
+    text-decoration: none;
+    color: white;
+    font-size: 17px;
+  }
+
+  .see-project2 {
+    border: 1px solid #ff6b00;
+    background: #ff6b00;
+    font-family: 'Inter', sans-serif;
+    margin-top: 377px;
+    height: 58px;
+    padding: 0%;
+  }
+
+  .project1 {
+    background: url(images/backgroundimg1.svg);
+  }
```

Comment:
- [ ] I know it's kind of confusing but if you pay closer attention in Figma design project-1 is like the rest of the projects but it has a hover-effect where when you hover over it the dark-background color is removed and title, description, and technologies is also removed
then you are left with `See Project`